src/createAnimatedComponent.tsx

```diff
@@ -311,13 +311,13 @@ export default function createAnimatedComponent(
     _getEventViewRef() {
       // Make sure to get the scrollable node for components that implement
       // `ScrollResponder.Mixin`.
-      return this._component?.getScrollableNode
-        ? this._component.getScrollableNode()
+      return (this._component as ComponentRef)?.getScrollableNode
+        ? (this._component as ComponentRef).getScrollableNode!()
```

Comment:
What about `.getScrollableNode?.()`?

Reply:
If I understood correctly, you want to change it to:

```jsx
      return (this._component as ComponentRef)?.getScrollableNode
        ? (this._component as ComponentRef).getScrollableNode?.()
        : this._component;
```
In that case I think we can keep non-null assertion, since the condition checks whether `getScrollableNode` exists. It would work if we could do something like:

```jsx
      return (this._component as ComponentRef).getScrollableNode?.();
```

however in that case we lose return value from else (`:`)

Reply:
I just wanted to get rid of the exclamation mark in `getScrollableNode!`.